### PR TITLE
Fixing `weighted_abs` feature position

### PR DESCRIPTION
**Pull request by freemansw1**
This PR addresses #148 and resolves a bug when using `weighted_abs` feature position in 1.3.x. Given the hotfix nature of this, I'm suggesting we stage it for a relatively rapidly released 1.3.2. 

* [x] Have you followed our guidelines in CONTRIBUTING.md? 
* [x] Have you self-reviewed your code and corrected any misspellings? 
* [ ] Have you written documentation that is easy to understand?
* [x] Have you written descriptive commit messages? 
* [ ] Have you added NumPy docstrings for newly added functions? 
* [x] Have you formatted your code using black? 
* [x] If you have introduced a new functionality, have you added adequate unit tests?
* [x] Have all tests passed in your local clone? 
* [ ] If you have introduced a new functionality, have you added an example notebook?
* [x] Have you kept your pull request small and limited so that it is easy to review? 
* [x] Have the newest changes from this branch been merged?

### CODE REVIEW

File: tobac/tests/test_feature_detection.py
Comment (JuliaKukulies) [inline tobac/tests/test_feature_detection.py:1]:
```suggestion

```

This import can probably removed then?

**Review by JuliaKukulies — APPROVED** [review]
Looks good - approved to merge and include in v1.3.2!

File: tobac/tests/test_feature_detection.py
Comment (freemansw1) [reply tobac/tests/test_feature_detection.py]: Yep, I've removed the extra imports and we now only import `tobac.testing` once.

**Comment by freemansw1** [comment]
Codecov seems to be having some problems today, but it looks like checks have passed. I'll merge if you're ok with it, @JuliaKukulies . I think once I merge this, I'll stage v1.3.2 for a release.

**Comment by JuliaKukulies** [comment]
I am OK with this!

**Pull request merged**
